src/mqt/ddsim/deterministicnoisesimulator.py

```diff
@@ -0,0 +1,27 @@
+"""Backend for DDSIM Unitary Simulator."""
+
+from __future__ import annotations
+
+from qiskit.transpiler import Target
+
+from .qasmsimulator import QasmSimulatorBackend
+
+
+class DeterministicNoiseSimulatorBackend(QasmSimulatorBackend):
+    """Python interface to MQT DDSIM deterministic noise-aware simulator."""
+
+    _SHOW_STATE_VECTOR = True
+    _HSF_SV_TARGET = Target(
+        description="MQT DDSIM Decision Diagram-based Noise-Aware Simulator Target",
+        num_qubits=QasmSimulatorBackend.max_qubits(),
+    )
+
+    def __init__(self) -> None:
+        super().__init__(
+            name="noise-aware_deterministic_decision-diagram_simulator",
```

Comment:
```suggestion
            name="density_matrix_dd_simulator",
```
Just a little shorter name that is still concise.

Reply:
I changed the name as suggested.